### DISCUSSION TITLE
Incorrect password. Unable to login with the password. No recovery key.

### DISCUSSION THREAD

**Discussion by DeveloperShiraz** [ente-io/ente#2808] [Q&A]
Unable to login. All I see is incorrect password for the password I created 2 minutes ago.

![Capture](https://github.com/user-attachments/assets/6e62caa5-3407-4a7a-886f-3366c18ad64c)

**Comment by ua741** [comment]
Hey @DeveloperShiraz , Can you email to support [at] ente.io with Subject `[Github Discussion 2808]`. I will need additional details, including the email which you used to sign up, and device details as well to debug this further.

**Comment by rhc94** [comment]
same thing.
same pass in android and web working and in win app "incorrect password". if i reset pass to same pass, then is ok, but i always need to reset pass and always use same pass...
